projenrc/github-workflows.ts

```diff
@@ -493,3 +493,152 @@ export function runCommitLintWorkflow(project: AwsCdkConstructLibrary) {
     }
   }
 }
+
+/**
+ * Runs the code generation step to update the list of available JumpStart models, and DLC images.
+ * @param project AwsCdkConstructLibrary
+ */
+export function buildCodeGenerationWorkflow(project: AwsCdkConstructLibrary) {
+
+  const CREATE_PATCH_STEP_ID = 'create_patch';
+  const PATCH_CREATED_OUTPUT = 'patch_created';
+
+  // First job to run the code generation step
+  const generate: Job = {
+    name: 'Code generation',
+    runsOn: ['ubuntu-latest'],
+    permissions: {
+      contents: JobPermission.READ,
+    },
+    outputs: {
+      [PATCH_CREATED_OUTPUT]: {
+        stepId: CREATE_PATCH_STEP_ID,
+        outputName: PATCH_CREATED_OUTPUT,
+      },
+    },
+    steps: [
+      {
+        name: 'Checkout project',
+        uses: 'actions/checkout@v3',
+        with: {
+          ref: 'main',
+        },
+      },
+      {
+        name: 'Setup Node.js',
+        uses: 'actions/setup-node@1a4442cacd436585916779262731d5b162bc6ec7',
```

Comment:
use a versioned version as we are swapping as needed in ../.projenrc.ts

Reply:
Done in latest version